_includes/footer.liquid

```diff
@@ -3,12 +3,17 @@
     {% include feedback.liquid %}
     <section>
       <div class="footer-logo">
-        {% img coat-of-arms.png alt: "Australian Government Coat of Arms" width:auto height:auto %}
+        {% img coat-of-arms.png alt:'Australian Government Coat of Arms' width:auto height:auto %}
       </div>
       <div class="footer-links">
         <nav>
           <ul>
-            <li><a href="mailto:guides@digital.gov.au">Email the Guides team</a></li>
+            <li>
+              <svg class="icon-inline fa-envelope-o" role="img" title="envelope email icon" aria-labelledby="fa-envelope-o-email">
```

Comment:
If you have two on the page you can make this a different ID so you don't double up. Maybe more secure than pointing to another code bit from another partial?